addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +41,63 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * @override
+  * Overrides values in addon/mixins/axis-titles.js
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
+    return this.get('graphicWidth') / 2;
+  }),
+
+  /**
+  * @override
+  * X Axis Titles needs some extra padding or else it will intersect with the lowest bar
+  * TO-DO: Change the 20px padding to be a variable that can be referenced elsewhere  
+  */
+  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return this.get('graphicBottom') + 20;
```

Comment:
I still have question about `20` number. We should figure out a variable name for that config